_tests_/port.test.js

```diff
@@ -0,0 +1,30 @@
+const Port = require("../src/Port.js");
```

Comment:
The filename should be written in PascalCase so it matches with the component it is testing